### ISSUE TITLE
[Question] Can LLaVa be pretrained to solve a multi-label classification task?

### ISSUE THREAD

**Issue by FHL1998** [haotian-liu/LLaVA#686]
### Question

I wonder if LLaVa can be used to solve a multi-label classification task, will it be better than separate image classification models? Do we still need to perform instruction pre-training and fine-tuning stages?
